boards/arduino/arduino_giga_r1/board.yml

```diff
@@ -0,0 +1,5 @@
+board:
+  name: arduino_giga_r1
+  vendor: arduino
+  socs:
+    - name: stm32h747xx
```

Comment:
The python script is buggy, the dash should be aligned to the socs 's' character

Reply:
So it has to be fixed in all stm32 based boards that have been merged till now. A clean up PR will be required. I propose at that point.

Reply:
> A clean up PR will be required. I propose at that point.

I'm fine with that, then we have a common cleanup PR for all STM.